### ISSUE TITLE
Add support for (more) log_slow_* settings

### ISSUE THREAD

**Issue by tersmitten** [Oefenweb/ansible-percona-server#7]

**Comment by mvdriel** [comment]
In #14 it is possible to create more flexible MySQL configuration. For example:

```
percona_server_my_cnf:
  - section: mysqld
    options:
      - name: log_slow_slave_statements
        value: 1
```

Does this fix this issue?

**Comment by tersmitten** [comment]
Fixed in #17

**Comment by tersmitten** [comment]
https://www.percona.com/doc/percona-server/5.5/diagnostics/slow_extended_55.html
